src/gui/qgsrelationeditorwidget.h

```diff
@@ -95,9 +95,11 @@ class GUI_EXPORT QgsRelationEditorWidget : public QgsAbstractRelationEditorWidge
 
     /**
      * Possible buttons shown in the relation editor
+     * \since QGIS 3.18
      */
     enum Button
     {
+      NoButton = 0, //!< No button
```

Comment:
Was the enum introduced in 3.18? If so, we'd need to add an extra (since 3.20) for NoButton to indicate it wasn't present in 3.18.